base/src/com/thoughtworks/go/util/GoConstants.java

```diff
@@ -58,7 +58,7 @@
 
     public static final String PRODUCT_NAME = "go";
 
-    public static final int CONFIG_SCHEMA_VERSION = 79;
+    public static final int CONFIG_SCHEMA_VERSION = 80;
```

Comment:
Just an FYI — you might conflict with WPC here. So I suggest bumping this to 81 and using @wpc's [cruise-config.xsd](https://github.com/gocd/gocd/pull/1887/files#diff-bdfabb73e2af551ffd8e50a573a66338) instead.

We'll make sure to merge yours after @wpc's or the other way around if @wpc prefers that.

Just let us know :)